sources/web/datalab/server.ts

```diff
@@ -94,14 +96,6 @@ function handleJupyterRequest(request: http.ServerRequest, response: http.Server
 function handleRequest(request: http.ServerRequest,
                        response: http.ServerResponse,
                        path: string) {
-  // TODO(jupyter): Additional custom path - should go away eventually with replaced
-  // pages.
-  // /static and /custom paths for returning static content
-  if ((path.indexOf('/static') == 0) || (path.indexOf('/custom') == 0)) {
```

Comment:
Then all static and custom requests are logged, which we try to avoid so it does not flood the log. Maybe add a if condition on logging.logRequest?